student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,7 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    //TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(my_username(), malloc_username_from_conf_file(), "success!");
```

Comment:
@mash9756 what happens to the string returned by `malloc_username_from_conf_file` here?

Reply:
It gets compared against the string returned by my_username(), and if they match, the message "success!" is returned
Following the Unity Assertion reference:
TEST_ASSERT_EQUAL_STRING (expected, actual)
Asserts that the null terminated ('\0')strings are identical. If strings are of different lengths or any portion of the strings before their terminators differ, the assertion fails. Two NULL strings (i.e. zero length) are considered equivalent.